app/controllers/user_controller.rb

```diff
@@ -3,6 +3,19 @@
 require 'bcrypt'
 
 class UserController < ApplicationController
+  def get_user_params
+    user_params = {
+      first_name: params[:first_name],
+      last_name: params[:last_name],
+      email: params[:email]
+    }
+    if params[:password]
+      password_match = /^(?=.*[a-zA-Z])(?=.*[0-9])(?=.*[\W]).{8,}$/.match?(params[:password])
```

Comment:
I thought we have this validation in the user model. Why do we need it here?

Reply:
I removed it. Was not supposed to be there.